.coveragerc

```diff
@@ -621,6 +621,9 @@ omit =
     homeassistant/components/izone/climate.py
     homeassistant/components/izone/discovery.py
     homeassistant/components/jellyfin/media_source.py
+    homeassistant/components/jewish_calendar/__init__.py
+    homeassistant/components/jewish_calendar/binary_sensor.py
+    homeassistant/components/jewish_calendar/sensor.py
```

Comment:
It is not allowed to remove existing tests

Reply:
I assumed so..